### DISCUSSION TITLE
Styling

### DISCUSSION THREAD

**Discussion by SMITTY-94** [Blazored/Modal#297] [Q&A] [answered]
I'm not sure if I missed this somewhere in the documentation.

How do I style the modals? I have it working properly but I cant seem to style the entire modal. I can target the inner elements but not the entire modal. For example I can't change the background-color.

Thanks!

**Comment by larsk2009** [comment, marked as answer]
Hey @AndrewCS149,

Have a look at [this section](https://github.com/Blazored/Modal#styling-the-modal) of the readme. Let me know if that is helpful or not. There is also a possibility to make your own complete custom modal if that is not enough.

**Reply by SMITTY-94** [reply]
The modal itself is great. Is there a way to simply change the background the color though?

**Reply by larsk2009** [reply]
Yeah sure, if you use the link I provided to apply a custom modal, you can just use normal CSS to style the modal.

**Reply by SMITTY-94** [reply]
![github](https://user-images.githubusercontent.com/52691521/110249021-bf4f5980-7f28-11eb-8775-e9facd88fcaa.jpg)

Im not sure I understand. What is the 'Class' being attached to? Im not grasping how im actually using this to connect to my own styling.

Also thanks a lot for the help!

**Reply by larsk2009** [reply]
The `Class` is set/attached to the Modal itself, specifically [this line](https://github.com/Blazored/Modal/blob/main/src/Blazored.Modal/BlazoredModalInstance.razor#L44). Hopefully that makes sense 😉.

**Reply by SMITTY-94** [reply]
Hmm Im still somewhat lost haha Would you be able to point me to some more concrete examples?

**Reply by larsk2009** [reply]
Sorry for taking so long to get back to you, stuff has been busy. I hope I have some time tomorrow or Friday to put together a working example.

**Reply by SMITTY-94** [reply]
No worries. That would be awesome if you can find the time. Please let me know and thanks a lot!

**Reply by larsk2009** [reply]
Well it took me a lot longer, but I didn't forget you! Have a look at [larsk2009/StyleExample](https://github.com/larsk2009/StyleExample) and let me know if it makes sense. It is a standard, VS2019 generated Blazor project where I only changed the minimum required to get a Modal with a custom class up and running.

**Reply by SMITTY-94** [reply]
Worked like a charm! Thanks man. I was definitely overthinking it haha Blazored Modal for the win!
